src/handlers/query.ts

```diff
@@ -277,6 +269,7 @@ function getSearchTextFilter(
       switch (expression.operator) {
         case "near_text":
         case "match_text":
+        case "hybrid_match_text":
           if (negated) {
             throw new Error("Negated near_text or match_text not supported");
```

Comment:
🙃 Would be nice to have `hybrid_match_text` included in the error message

(can't comment there but ditto on line 277)